packages/core/js-client/src/index.ts

```diff
@@ -29,7 +29,7 @@ import { BlobWorker, Worker } from "threads/master";
 import { ClientPeer, makeClientPeerConfig } from "./clientPeer/ClientPeer.js";
 import { callAquaFunction } from "./compilerSupport/callFunction.js";
 import { registerService } from "./compilerSupport/registerService.js";
-import { fetchResource } from "./fetchers/index.js";
+import { fetchResource } from "#fetcher";
```

Comment:
Don't you need some stuff in package.json for this to work?

Reply:
i already merged that stuff in master